### DISCUSSION TITLE
Java 15 requirement

### DISCUSSION THREAD

**Discussion by IlyaMar** [hierynomus/sshj#728] [General]
Hello,

I see in depends bcprov-jdk15on:1.68 that has class file version 59, i e compiled for Java 15.
Does it mean that sshj requires Java 15? In what scenarios this dependency is used?

**Comment by vladimirlagunov** [comment]
Hi! It's Java 1.5, not Java 15.
https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on

**Reply by IlyaMar** [reply]
Ok I've found that bcprov library has multi-release jar and have classes compiled under several java versions under META-INF/versions folder. There are classes for 11 and 15 java.
Problem was my integration tests have jacoco byte-code instrumentation for coverage and I use java 11.
